specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
@sfsinger19103 and I were talking about annotating the domain model to highlight which concepts are necessary for the RLA Tool that we are building for this contract, and which ones are there for other reasons.  E.g., some terms are there because they are part of the domain model of elections in the USA.  Others are there because they are mentioned in the C.R.S. but are not (at the moment) mentioned in the Section on RLAs.  What do you all think about such?  I intend to write a paragraph about this in the preface anyway (there is a `@todo` for such), but highlighting them in an extra special fashion (like special coloring, margin comments or markers, etc.) is another more detailed way to go. @mkiniry can help with this polishing, even as a part of his current work in the `glossary-specification-double-check` branch.